test/core/imports.wast

```diff
@@ -13,9 +13,12 @@
   (global (export "global-mut-i64") (mut i64) (i64.const 66))
   (table (export "table-10-inf") 10 funcref)
   (table (export "table-10-20") 10 20 funcref)
+  (table (export "table64-10-inf") i64 10 funcref)
+  (table (export "table64-10-20") i64 10 20 funcref)
   (memory (export "memory-2-inf") 2)
-  ;; Multiple memories are not yet supported
-  ;; (memory (export "memory-2-4") 2 4)
```

Comment:
This seems like something that should maybe be done over in them multi-memory proposal? .. but perhaps not worth splitting up at this point?

Reply:
In my opinion it clearly should have been done in the multi-memory process, but multi-memory is already [stage 5](https://webassembly.org/features/). That said, it seems like the multi-memory proposal still hasn't been merged into the main spec repo - not even into the wasm-3.0 branch? (Perhaps @rossberg can confirm. This seems like an oversight to me.)

In any case, I would rather just keep it all here since it's just tests and this is hopefully about to become standard as well anyway.

Reply:
Multi-memory definitely is included in the wasm-3.0 branch. It comes with quite a few tests, especially in the multi-memory subdirectory.

IIRC, the above lines long predate the actual work on that proposal, so it was an oversight on my part not to activate them, too. I don't mind having that happen in this PR. :)

Reply:
In that case, expect a PR from me to fix an issue in module validation 😄